src/Curation/Curation.router.ts

```diff
@@ -191,6 +195,20 @@ export class CurationRouter extends Router {
 
   getCollectionItemCurations = async (req: AuthRequest) => {
     const collectionId = server.extractFromReq(req, 'id')
+    let itemIds: string[] | undefined
+    try {
+      itemIds = server.extractFromReq(req, 'itemIds')
+    } catch (error) {}
+
+    if (itemIds && !Array.isArray(itemIds)) {
```

Comment:
There are no tests for the pagination logic for the curations, would you mind adding them?